metrics/metrics.go

```diff
@@ -255,6 +256,11 @@ var (
 		Help: "Timestamp when the drand process started up in seconds since the Epoch",
 	})
 
+	ErrorSendingPartialCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Why name this a counter if you're using it as a round indicator?

Reply:
It's not really a round indicator, it's just there as additional metadata, the crux is that we receive a `(time, count, round, address)` tuple and we can group by the address to work out how many parties are down for each time period